Source/WebCore/Modules/mediastream/gstreamer/GStreamerPeerConnectionBackend.cpp

```diff
@@ -69,9 +69,24 @@ GStreamerPeerConnectionBackend::GStreamerPeerConnectionBackend(RTCPeerConnection
     , m_endpoint(GStreamerMediaEndpoint::create(*this))
 {
     disableICECandidateFiltering();
+    logger().addObserver(*this);
 }
 
-GStreamerPeerConnectionBackend::~GStreamerPeerConnectionBackend() = default;
+GStreamerPeerConnectionBackend::~GStreamerPeerConnectionBackend()
+{
+    logger().removeObserver(*this);
+}
+
+void GStreamerPeerConnectionBackend::didLogMessage(const WTFLogChannel&, WTFLogLevel, Vector<JSONLogValue>&& values)
+{
+#ifndef GST_DISABLE_GST_DEBUG
+    StringBuilder builder;
+    for (auto& [type, value] : values)
```

Comment:
Sorry that I didn't mention this before, but if you want to deconstruct and not using the first field, you can just use _